app/lib/system/deploy.rb

```diff
@@ -26,6 +26,11 @@ def initialize(info)
         @error = info.fetch(:error) if info.key?(:error)
       end
 
+      def docs_version
```

Comment:
Moving it here to group all of the version-related stuff (which is extremely confusing) in one file to avoid jumping around.

Also, memoizing - we don't need to calculate this on every request.